### PR TITLE
Add Bulgarian translations

### PR DESCRIPTION
**Pull request by kbkozlev**
**Summary**

I have added a Bulgarian translation

**New feature checklist**

- [x] I have ran the application to make sure my code runs
- [x] This not just a simple formatting change

**Translation checklist**

- [x] I have ran the application to make sure my code runs
- [x] I have used the correct [ISO 639-1 code](https://www.alchemysoftware.com/livedocs/ezscript/Topics/Catalyst/Language.htm) for my translations (e.g. `en`) 
- [x] I have made sure to leave a `,` at the end of each translation added
- [x] I have added the language to `supportedLanguages` in alphabetical order
- [x] I have added the language to the table in README.md in alphabetical order
- [x] (*If you translated the README*) I have translated the README and linked it in the main README.md

### CODE REVIEW

File: auto_py_to_exe/web/js/i18n.js [hunk 1]
```diff
@@ -114,6 +117,8 @@ const translationMap = {
                 vi: 'Cửa sổ bảng điều khiển',
                 nl: 'Console Venster',
                 ko: "콘솔 화면",
+                bg: "Конзолно приложение",
+
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:121]: Can drop this extra line

File: README-Bulgarian.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+| Thai (ภาษาไทย)                              | [teerut26](https://github.com/teerut26)                                         | UI (partial)                         |
+| Turkish (Türkçe)                            | [mcagriaksoy](https://github.com/mcagriaksoy)                                   | UI and [README](./README-Turkish.md) |
+| Ukrainian (Українська)                      | [AndrejGorodnij](https://github.com/AndrejGorodnij)                             | UI                                   |
+| Vietnamese (Tiếng Việt)                     | [7777Hecker](https://github.com/7777Hecker)                                     | UI                                   |
+| Bulgarian (Български)                       | [kbkozlev](https://github.com/kbkozlev)                                         | UI and [README](README-Bulgarian.md) |
+
+> Искате да добавите превод на друг език? Актуализирайте [i18n.js](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/auto_py_to_exe/web/js/i18n.js) и представете заявка за промяна (PR) или го прикачете към проблем (issue).
```
Comment (brentvollebregt) [inline README-Bulgarian.md:146]: You can leave this table out - just since it changes quite a bit

**Review by brentvollebregt — CHANGES_REQUESTED** [review]
Looking good! Just a couple of small things and then I can put this in.

**Comment by kbkozlev** [comment]
I have made the changes.

**Review by brentvollebregt — APPROVED** [review]
Excellent, thank you 👍

**Comment by brentvollebregt** [comment]
Thanks for the contribution! This has been released in [v2.42.0](https://github.com/brentvollebregt/auto-py-to-exe/releases/tag/v2.42.0) 🚀

**Pull request merged**